packages/fxa-settings/src/components/CheckboxBlue/index.tsx

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ChangeEvent, useState, useCallback } from 'react';
+import classNames from 'classnames';
+import { ReactComponent as CheckWhite } from './icon-check-white.svg';
+
+export type CheckboxBlueProps = {
+  defaultChecked?: boolean;
+  disabled?: boolean;
+  label?: string;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  prefixDataTestId?: string;
+};
+
+// This CheckboxBlue component is modeled on the most recently updated styles for non-authenticated flows
```

Comment:
I haven't checked about removing the existing component yet - figured we could make sure this one worked properly first then compare. Retrofitting the existing checkbox component to accommodate these styles seemed a bit messy since I wasn't sure if the previous `Checkbox` should be discontinued entirely, so I decided to keep them separate for now.